rhalphalib/util.py

```diff
@@ -13,13 +13,13 @@ def _to_numpy(hinput):
             raise ValueError("Counts array and binning array are incompatible in tuple %r" % hinput)
         return hinput
     elif str(type(hinput)) == "<class 'ROOT.TH1D'>":
-        sumw = np.zeros(hinput.GetNBinsX())
+        sumw = np.zeros(hinput.GetNbinsX())
         binning = np.zeros(sumw.size + 1)
         name = hinput.GetName()
         for i in range(1, sumw.size + 1):
-            sumw[i] = hinput.GetBinContent(i)
-            binning[i] = hinput.GetXaxis().GetBinLowEdge(i)
-        binning[i+1] = hinput.GetXaxis().GetBinUpEdge(i)
+            sumw[i-1] = hinput.GetBinContent(i)
+            binning[i-1] = hinput.GetXaxis().GetBinLowEdge(i)
+            binning[i] = hinput.GetXaxis().GetBinUpEdge(i)
```

Comment:
In principle this line could have the original indent, because the variable `i` will be set to the last value as it exits the loop and then the last bin edge gets set.  At present, it sets all the interior `binning[1:-1]` bins twice, albeit to the same value.